modules/fbs-core/web/src/app/page-components/sidebar/sidebar.component.html

```diff
@@ -86,6 +86,10 @@ <h1>
         <mat-icon>account_circle</mat-icon>
       </button>
       <mat-menu #menu="matMenu">
+        <button mat-menu-item>
+          <mat-icon class="person-icon">person</mat-icon>
+          {{ user.prename + " " + user.surname }}
```

Comment:
```suggestion
          {{ user.prename }} {{ user.surname }}
```

You don't need to concatenate strings with the plus symbol if you put both variables in parentheses and leave a space between them. This is a bit nicer